vm/vm.go

```diff
@@ -138,6 +139,12 @@ func JunoAppendDataGasConsumed(readerHandle C.uintptr_t, ptr unsafe.Pointer) {
 	context.dataGasConsumed = append(context.dataGasConsumed, makeFeltFromPtr(ptr))
 }
 
+//export JunoAddExecutionSteps
+func JunoAddExecutionSteps(readerHandle C.uintptr_t, execSteps C.ulonglong) {
+	context := unwrapContext(readerHandle)
+	context.executionSteps += uint64(execSteps) // assumes initial default value is zero even in ffi
```

Comment:
so i added this earlier to make sure (I checked later) that the initial values on object creation is zero(0) and not some random data in memory, since it was being called over ffi.
I didn't want to assume the variable would be initialized to zero just because go does initialize its variables on creation.

Its irrelevant now